src/WorkflowGuiBundle/DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
```

Comment:
please add my header here

Reply:
Oh... thats not good :D, Ok. Lets do your PR without headers, and I will add them afterwards

Reply:
Do you mean header from `src/WorkflowGuiBundle/Resources/public/js/workflow/item.js` for instance?

Reply:
yes, that would be the right one.